src/test/java/guitests/AddCommandTest.java

```diff
@@ -1,53 +1,64 @@
-//
-//public class AddCommandTest extends AddressBookGuiTest {
-//
-//    @Test
-//    public void add() {
-//        //add one person
-//        TestPerson[] currentList = td.getTypicalPersons();
-//        TestPerson personToAdd = td.hoon;
-//        assertAddSuccess(personToAdd, currentList);
-//        currentList = TestUtil.addPersonsToList(currentList, personToAdd);
-//
-//        //add another person
-//        personToAdd = td.ida;
-//        assertAddSuccess(personToAdd, currentList);
-//        currentList = TestUtil.addPersonsToList(currentList, personToAdd);
-//
-//        //add duplicate person
-//        commandBox.runCommand(td.hoon.getAddCommand());
-//        assertResultMessage(AddCommand.MESSAGE_DUPLICATE_PERSON);
-//        assertTrue(personListPanel.isListMatching(currentList));
-//
-//        //add to empty list
-//        commandBox.runCommand("clear");
-//        assertAddSuccess(td.alice);
-//
-//        //invalid command
-//        commandBox.runCommand("adds Johnny");
-//        assertResultMessage(Messages.UNKNOWN_COMMAND);
-//    }
-//
-//    private void assertAddSuccess(TestPerson personToAdd, TestPerson... currentList) {
-//        commandBox.runCommand(personToAdd.getAddCommand());
-//
-//        //confirm the new card contains the right data
-//        PersonCardHandle addedCard = personListPanel.navigateToPerson(personToAdd.getName().fullName);
-//        assertMatching(personToAdd, addedCard);
-//
-//        //confirm the list now contains all previous persons plus the new person
-//        TestPerson[] expectedList = TestUtil.addPersonsToList(currentList, personToAdd);
-//        assertTrue(personListPanel.isListMatching(expectedList));
-//    }
-//
-//}
+package guitests;
+
+import seedu.commando.commons.core.Messages;
+import seedu.commando.logic.commands.AddCommand;
+import seedu.commando.model.todo.ToDo;
+
+import org.junit.Test;
+
+import guitests.guihandles.EventCardHandle;
+import guitests.guihandles.ToDoCardHandle;
+import guitests.utils.CommandBuilder;
+import guitests.utils.TestUtil;
+
+import static org.junit.Assert.assertTrue;
+
+public class AddCommandTest extends CommanDoGuiTest {
+
+    @Test
+    public void add() {
+        commandBox.runCommand("clear");
+        //add one todo
+        ToDo[] currentList = td.getEmptyToDos();
+        ToDo toDoToAdd = td.toDoItem2;
+        assertAddSuccess(toDoToAdd, currentList);
+        currentList = TestUtil.addToDosToList(currentList, toDoToAdd);
+
+        //add another todo
+        toDoToAdd = td.toDoItem1;
+        assertAddSuccess(toDoToAdd, currentList);
+        currentList = TestUtil.addToDosToList(currentList, toDoToAdd);
+
+        //add duplicate person
+        commandBox.runCommand(CommandBuilder.buildAddCommand(td.toDoItem1));
+        assertResultMessage(Messages.TODO_ALREADY_EXISTS);
+        assertTrue(eventListPanel.isListMatching(currentList) && taskListPanel.isListMatching(eventListPanel.getListViewSize(), currentList));
```

Comment:
You created isBothListMatching method in the previous commit. I assume you are going to abstract this part later on.